src/Module/Login.php

```diff
@@ -414,20 +414,22 @@ private static function getRegisterURL()
 			}
 		}
 
+		$arg = [];
```

Comment:
Please rename to `$args` or `$arg_list` to denote the array type.

Reply:
args :-)

Reply:
done